packages/lit-dev-content/site/tutorials/content/svg-templates/00.md

```diff
@@ -0,0 +1,14 @@
+SVG enables artists to create scalable graphics through an XML-like syntax. It's supported in most modern browsers.
+
+But beautiful graphics can get complicated!
+
+Webcomponents are a perfect match for SVG. They can encapsulate styles,
+logic, and document structure.
+
+Lit can maintain SVG through tempaltes and template fragments.
+They provide a declarative way to programmatically generate complex
+graphics.
+
+After completing this tutorial, the reader will be familiar with
```

Comment:
In this tutorial You will:

- learn the basics of rendering svgs with lit
- build X
- learn y

Reply:
sounds good! done